pages/docs/tools/index.html

```diff
@@ -0,0 +1,65 @@
+title: Tools Overview
+description: Management tools, utilities and data visualization for Crate
+author: Christian Haudum
+
+
+There are several tools and utilities that work with Crate.
```

Comment:
I think this first "paragraph" can be improved, something like:

there are various different tools and utilities that can connect to crate, ranging from ... to .... 
they can do this and that, ... use JDBC standard .... HTTP

I think you know what I mean :)